src/jetpack-editor-setup.js

```diff
@@ -66,15 +62,6 @@ export function setupJetpackEditor( jetpackState ) {
 	return setJetpackData( jetpackState );
 }
 
-export function setupJetpackLocale( locale, extraTranslations ) {
-	setupLocale( {
-		domain: JETPACK_LOCALE_DOMAIN,
-		locale,
-		extraTranslations,
-		getTranslation: getJetpackTranslation,
-	} );
-}
```

Comment:
The translations for Jetpack are now included in the index file when registering Gutenberg ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/504270493676fe9c2a219519bb70a8d76106dee1/src/index.js#L12-L21)).